src/Resolver/Service.php

```diff
@@ -49,17 +49,30 @@ public function resolve($definition)
             throw new \InvalidArgumentException('$definition must be an instance of ' . Definition::class);
         }
 
-        $url           = $this->getIterator()->get('url', $definition);
-        $query         = $this->getIterator()->get('query', $definition);
-        $method        = $definition->has('method') ? $this->getIterator()->get('method', $definition) : 'POST';
-        $headers       = $definition->has('headers') ? $this->getIterator()->get('headers', $definition) : [];
-        $variables     = $definition->has('variables') ? $this->getIterator()->get('variables', $definition) : [];
+        $headers = ['Content-type' => 'application/json'];
+
+        $url             = $this->getIterator()->get('url', $definition);
+        $query           = $this->getIterator()->get('query', $definition);
+        $method          = $definition->has('method') ? $this->getIterator()->get('method', $definition) : 'POST';
+        $variables       = $definition->has('variables') ? $this->getIterator()->get('variables', $definition) : [];
+        $ignoreSSLErrors = $definition->has('ignoreSSLErrors')
+            ? $this->getIterator()->get('ignoreSSLErrors', $definition)
+            : false;
+        $headers = $definition->has('headers')
```

Comment:
We should move applying this header down to the method logic gate below; only apply to POST requests.

From a spec standpoint, I think I would rather see this in config, but I suppose this reduces boilerplate, which is one of the tailing objectives at the end of the spec. Fine leaving for now.

Reply:
I totally agree on preferring an application wide config over a request-by-request setup we have now. @zetlen and I discussed this briefly as well last week.